legate/tester/__init__.py

```diff
@@ -91,3 +91,6 @@ class CustomTest:
 #:
 #: Client test scripts should udpate this set with their own customizations.
 CUSTOM_FILES: list[CustomTest] = []
+
+#: Location to store a list of last-failed tests
+LAST_FAILED_FILENAME: str = ".legate-test-last-failed"
```

Comment:
We could e.g. set this to `".cununeric-test-last-failed"` in cunumeric's `test.py`